pkgs/by-name/re/readstat/package.nix

```diff
@@ -36,12 +45,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     pkg-config
     autoreconfHook
+    libtool
   ];
 
-  buildInputs = [ libiconv ];
-
-  enableParallelBuilding = true;
-
```

Comment:
why was this removed?

Reply:
libiconv wasn't actually used.

I thought that we didn't need the parallel building. I'll restore it.

Reply:
I was not asking for restoring it, I don't even know why it is required. I was merely asking the reasons why it was removed.

Reply:
> I was not asking for restoring it, I don't even know why it is required. I was merely asking the reasons why it was removed.

re: libiconv, the boilerplate supplied by autoconf apparently required gettext and libiconv, but those features were never actually used. Since I removed teh requirement on gettext (in the patch), libiconv came away with it.